<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Game quay số:</string>
    </property>
    <widget class="QLabel" name="labelSo1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>71</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(221, 255, 255);
color: rgb(255, 0, 127);</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelSo2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>71</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(221, 255, 255);
color: rgb(255, 0, 127);</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelSo3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>71</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(221, 255, 255);
color: rgb(255, 0, 127);</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Tiền máy: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTienMay">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTienNguoi">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>210</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Tiền người:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonQuaySo">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>260</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Quay số</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonGameMoi">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>260</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Game mới</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonKetThuc">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>260</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Kết thúc</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>503</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
